lib/tests/2wp.js

```diff
@@ -139,6 +207,34 @@ const assert2wpBalancesAfterSuccessfulPegin = async (initial2wpBalances, peginVa
 
 };
 
+const addInputs = (tx, senderUtxosInfo) => {
```

Comment:
```suggestion
const addInputs = (tx, utxos) => {
```
Maybe that simplifies? No need to receive the whole object

Reply:
Updated.